djangocms_versioning_filer/templates/admin/filer/base_site.html

```diff
@@ -0,0 +1,43 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}
+    {{ block.super }}
+
+    <link rel="stylesheet" type="text/css" href="{% static 'filer/css/admin_filer.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'filer/css/micromodal.css' %}">
+{% endblock %}
+
+{% block extrahead %}
+    {{ block.super }}
+
+    {# upload stuff #}
+    {{ media.js }}
+    <script type="text/javascript">
+        var __jQuery = django.jQuery;
+        window.jQuery = (__jQuery) ? __jQuery : window.jQuery || undefined;
+        window.$ = window.jQuery;
+    </script>
+    <script src="{% static 'filer/js/libs/class.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/jquery-ui.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/dropzone.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/mediator.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/jquery.cookie.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/fileuploader.min.js' %}"></script>
+    <script src="{% static 'admin/js/admin/RelatedObjectLookups.js' %}"></script>
+    <script src="{% static 'filer/js/addons/dropdown-menu.js' %}"></script>
+    <script src="{% static 'filer/js/addons/popup_handling.js' %}"></script>
+    <script src="{% static 'filer/js/addons/micromodal.min.js' %}"></script>
```

Comment:
I don't think that we need base site, you can add to the extra head the same as: [djangocms_versioning_filer/templates/admin/filer/folder/directory_listing.html](https://github.com/divio/djangocms-versioning-filer/pull/78/files#diff-b1fc33cb8664c5c58ab277724fcc74904e9b2baeeb3b0ad1a1baa4144698a872)

Reply:
@Aiky30 Reason for adding the base_site is micromodal.min.js needs to be loaded before table-dropzone.js to ensure the micromodal instance is created
 
https://github.com/Bernardvdv/djangocms-versioning-filer/blob/feat/dialog_with_file_constraints/djangocms_versioning_filer/static/filer/js/addons/table-dropzone.js#L13

Reply:
@Bernardvdv It can be added before the {{ block.super }} tag.

Reply:
@Aiky30 Done, thanks